test/integration/fixtures/uncaught/listeners.fixture.js

```diff
@@ -3,7 +3,8 @@
 const assert = require('assert');
 const mocha = require("../../../../lib/mocha");
 
-for (let i = 0; i < 15; i++) {
+// keep this low to avoid warning
+for (let i = 0; i < 5; i++) {
```

Comment:
@juergba 

the max listener limit is 10, so node was dumping 'max listeners exceeded' warnings.  this went unnoticed because--by default--the integration tests would pipe `STDERR` into `/dev/null`.  with the changes in this PR, `STDERR` is output to terminal, _unless_ the test is configured (via `{stdio: 'pipe'}`) to trap the output so it can be used in an assertion.

Reply:
sorry, the changes in #4241 surfaced this.